<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="BHAVESH" ignored="0" name="BankingWebsite" tests="7" failures="3" timestamp="17 Jun 2022 19:46:02 GMT" time="77.197" errors="0">
  <testcase name="loginTest" time="3.235" classname="com.inetBanking.testCases.TC_LoginTest_001"/>
  <testcase name="LoginDataTest" time="3.435" classname="com.inetBanking.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginDDT_002.LoginDataTest(TC_LoginDDT_002.java:31)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- LoginDataTest -->
  <testcase name="LoginDataTest" time="7.015" classname="com.inetBanking.testCases.TC_LoginDDT_002"/>
  <testcase name="LoginDataTest" time="3.027" classname="com.inetBanking.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginDDT_002.LoginDataTest(TC_LoginDDT_002.java:31)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- LoginDataTest -->
  <testcase name="LoginDataTest" time="4.492" classname="com.inetBanking.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginDDT_002.LoginDataTest(TC_LoginDDT_002.java:31)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- LoginDataTest -->
  <testcase name="LoginDataTest" time="6.471" classname="com.inetBanking.testCases.TC_LoginDDT_002"/>
  <testcase name="AddCustomer" time="10.47" classname="com.inetBanking.testCases.TC_AddCustomer_003"/>
</testsuite> <!-- BankingWebsite -->
